website/src/docs/hotchocolate/filtering/comparable-filter.md

```diff
@@ -0,0 +1,81 @@
+### Comparable Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.